wca-regulations.md

```diff
@@ -401,9 +401,9 @@ Note: Because Article and Regulation numbers are not reassigned when Regulations
     - A6b) The competitor is responsible for stopping the Stackmat timer correctly.
         - A6b1) If the timer stops before the end of the solve and the timer shows a time strictly below 0.06 seconds, then the attempt is replaced by an extra attempt. A competitor forfeits the right to the additional attempt if the WCA Delegate determines that the timer was stopped deliberately.
         - A6b2) If the timer stops before the end of the solve and displays a time of 0.06 seconds or higher, then the attempt is disqualified (DNF). Exception: if the competitor can demonstrate that the timer malfunctioned, they may receive an extra attempt, at the discretion of the WCA Delegate.
-    - A6c) The competitor must fully release the puzzle before stopping the solve. Penalty: time penalty (+2 seconds).
+    - A6c) The competitor must fully release the puzzle before stopping the solve. Penalty: disqualification of the attempt (DNF). Exception: If no moves have been applied after stopping the solve and before releasing the puzzle, a time penalty (+2 seconds) may be assigned instead, at the discretion of the judge.
```

Comment:
Given the phrasing of A6a (`The competitor stops the solve by releasing the puzzle and then stopping the timer`), maybe change references to `stopping the solve` to `stopping the timer`?  I could see possible confusion where (a very pedantic) someone gets stuck in a recursive loop :P

Reply:
How would you loop?

I was about to disagree with the suggestion, but while writing the explanation below I realized it's actually better to use "stop the timer".

The way I see it, the attempt has 3 states: inspection, solve, recording.
Article A (3 to 7) describes what you can do during these states, and how transitioning between them works.

"Stopping the solve" is the transition between "solve" and "recording", and consist of two actions: "releasing the puzzle" and "stopping the timer".
A6c describes what happens when you mess up the order of the actions **within** the transition, so it doesn't make much sense to refer to "after/before stopping the solve", when you're precisely describing what is "stopping the solve".

So basically I would indeed replace occurrences of "stopping the solve" by "stopping the timer" within A6c.